### PR TITLE
en: add handle-alerts.md and alert-rules.md

### PR DESCRIPTION
**Pull request by Damon-PingCap**
add handle-alerts.md and alert-rules.md

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/93
- Other reference link(s):

### CODE REVIEW

File: en/handle-alerts.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+This document introduces how to deal with the major alert information in DM.
+
+## The alert of task status
+
+### `DM_task_state`
+
+The alert triggered when the DM-worker has sub-task in `Paused` status for over 20 minutes. You need to refer to [DM error handling](error-handling.md#Data-Migration-Error-Handling).
+
+## The alert of relay log
```
Comment (toutdesuite) [inline en/handle-alerts.md:17]:
```suggestion
## Alert rules related to relay log
```

File: en/handle-alerts.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+### `DM_relay_process_exits_with_error`
```
Comment (toutdesuite) [inline en/handle-alerts.md:19]: Please update other alert rules accordingly. `When something happens, an alert is triggered `~
Please update changes in `zh/monitor-a-dm-cluster.md` & `zh/replicate-data-using-dm.md` in https://github.com/pingcap/docs-dm/pull/93/files, thanks~

**Comment by Damon-PingCap** [comment]
PTAL

**Comment by WangXiangUSTC** [comment]
ci failed because had a dead link, please fix it

File: en/alert-rules.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: DM Alert Information
+summary: Introduce the alert information of DM.
+category: reference
```
Comment (toutdesuite) [inline en/alert-rules.md:4]:
```suggestion
```

File: en/alert-rules.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+# DM Alert Information
+
+The [alert system](replicate-data-using-dm.md#step-7-monitor-the-task-and-check-logs) is deployed by default when you deploy a DM cluster using DM-Ansible.
+
+> **Note:**
+>
+> There are alert rules provided with DM-worker only. But it does not include DM-master currently.
```
Comment (toutdesuite) [inline en/alert-rules.md:13]:
```suggestion
> There are alert rules provided with DM-worker only.
```

File: en/handle-alerts.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+When the relay log processing unit has some error, an alert is triggered and the relay log processing unit's status becomes to `Paused`. You need to refer to [DM error handling](error-handling.md#Data-Migration-Error-Handling).
+
+### `DM_remain_storage_of_relay_log`
+
+When the disk free space of relay log was less than 10G, an alert is triggered. You can refer the handling method below.
+
+- Delete unwanted data by manual to increase disk free space.
+- Try to adjust the [automatic data purge policy of relay log](relay-log.md#Automatic data purge) or execute [manual data purge](relay-log.md#Manual-data-purge).
```
Comment (toutdesuite) [inline en/handle-alerts.md:28]:
```suggestion
- Reconfigure the [automatic data purge strategy of the relay log](relay-log.md#automatic-data-purge) or[purge data manually](relay-log.md#manual-data-purge).
```

File: en/handle-alerts.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+- Try to [migrate DM-worker instance](cluster-operations.md#Replace/migrate-a-DM-master-instance) to another disk which has enough free space.
```
Comment (toutdesuite) [inline en/handle-alerts.md:29]:
```suggestion
- [Migrate the DM-worker instance](cluster-operations.md#replacemigrate-a-dm-worker-instance) to a disk with enough free space.
```

File: en/handle-alerts.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+### `DM_binlog_file_gap_between_master_relay`
+
+When the number of latest binlog files extracted by the relay log processing unit lags behind the current upstream MySQL/MariaDB number by 1 (excluding 1) for over 10 minutes, an alert is triggered. You need to refer to [DM error handling](error-handling.md#Data-Migration-Error-Handling).
+
+## Alert rules related to Dump/Load
+
+### `DM_dump_process_exists_with_error`
+
+When the dump processing unit has some error, an alert is triggered and the dump processing unit'status becomes to `Paused`. You need to refer to [DM error handling](error-handling.md#Data-Migration-Error-Handling).
```
Comment (toutdesuite) [inline en/handle-alerts.md:51]:
```suggestion
When the Dump processing unit encounters an error, an alert is triggered and this unit moves to the `Paused` status. You need to refer to [Troubleshoot DM](error-handling.md#troubleshooting).
```

File: en/handle-alerts.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+### `DM_load_process_exists_with_error`
+
+When the load processing unit has some error, an alert is triggered and the load processing unit'status becomes to `Paused`. You need to refer to [DM error handling](error-handling.md#Data-Migration-Error-Handling).
```
Comment (toutdesuite) [inline en/handle-alerts.md:55]:
```suggestion
When the Load processing unit encounters an error, an alert is triggered and this unit moves to the `Paused` state. You need to refer to [Troubleshoot DM](error-handling.md#troubleshooting).
```

File: en/handle-alerts.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+## Aler rules related to binlog replication
```
Comment (toutdesuite) [inline en/handle-alerts.md:57]:
```suggestion
## Alert rules related to binlog replication
```

**Comment by toutdesuite** [comment]
TOC updated in https://github.com/pingcap/docs-dm/pull/133

**Pull request merged**